assets/javascripts/discourse/connectors/extra-nav-item/activity-pub-navigation.gjs

```diff
@@ -0,0 +1,15 @@
+import Component from "@ember/component";
+import { classNames, tagName } from "@ember-decorators/component";
+import ActivityPubNavItem from "../../components/activity-pub-nav-item";
+
+@tagName("span")
```

Comment:
```suggestion
@tagName("li")
```

Otherwise this shows up as: 

![CleanShot 2025-06-05 at 10 08 12@2x](https://github.com/user-attachments/assets/d10dfa36-cb0c-46d2-8ca4-dca9f3b64921)